### PR TITLE
PLANNER-1373: Display Connection error when WS is down

### PR DESCRIPTION
**Pull request by danielefiungo**
Merge request related to #28

### CODE REVIEW

File: optaweb-tsp-ui/src/components/LocationList.tsx [hunk 1]
```diff
@@ -90,7 +90,10 @@ const LocationList: React.SFC<ILocationListProps> = (
   props: ILocationListProps
 ) => {
   return (
-    <div className="leaflet-top leaflet-left leaflet-touch">
+    <div
+      className="leaflet-top leaflet-left leaflet-touch"
+      style={{ zIndex: 500 }}
```
[the remark this answers is not in the document]
Comment (danielefiungo) [reply optaweb-tsp-ui/src/components/LocationList.tsx]: Yes, I'll remove it

File: optaweb-tsp-ui/src/components/ConnectionError.tsx [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+      <Text component={TextVariants.small}>{this.props.help}</Text>
+    ) : (
+      ''
+    );
+  }
+
+  render() {
+    const { title, message, icon } = this.props;
+    return (
+      <Modal title={title} isOpen={true} style={{ zIndex: 1000000 }}>
```
[the remark this answers is not in the document]
Comment (danielefiungo) [reply optaweb-tsp-ui/src/components/ConnectionError.tsx]: Right

File: optaweb-tsp-ui/src/containers/App.tsx [hunk 3]
```diff
@@ -54,16 +57,27 @@ class App extends Component<IAppProps> {
...
     }
   }
 
   render() {
+    const { ws } = this.props.tsp;
     return (
       <div>
+        {ws === types.WS_CONNECTION_STATE.ERROR && (
+          <ConnectionError
+            title="Ops... Connection Error!!!"
```
Comment (yurloc) [inline optaweb-tsp-ui/src/containers/App.tsx]:
```suggestion
            title="Oops... Connection error!"
```

File: optaweb-tsp-ui/src/containers/App.tsx [hunk 4]
```diff
@@ -54,16 +57,27 @@ class App extends Component<IAppProps> {
...
+            message="Please check your network connection?"
```
Comment (yurloc) [inline optaweb-tsp-ui/src/containers/App.tsx]:
```suggestion
            message="Please check your network connection."
```

File: optaweb-tsp-ui/src/containers/App.tsx [hunk 5]
```diff
@@ -54,16 +57,27 @@ class App extends Component<IAppProps> {
...
+            icon={<UnpluggedIcon />}
+            help={
+              'When connection will be available the application will be funcional again.'
```
Comment (yurloc) [inline optaweb-tsp-ui/src/containers/App.tsx]:
```suggestion
              'When connection is available the application will be functional again.'
```

**Pull request merged**